coverage_assessment.R

```diff
@@ -41,7 +41,118 @@ nut <- get_data()
 
 # Problem 3: Prevalence of acute undernutrition/malnutrition ----
 
+library(dplyr)
+
+## Load functions in R directory ----
+for (f in list.files("R", full.names = TRUE)) source (f)
+
+
+# Read data ----
+nut <- get_data()
+
+
+####MAM
+
+with(nut, sum(muac >= 11.5 & muac <12.5  & age >=6 & age <=59 & oedema == 2, na.rm= TRUE))
+
+##number of moderate acute malnourished (MAM) cases children 6-59 months old 
+
+Male_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 & age >=6 & age <=59 & oedema == 2 & sex == 1 , na.rm= TRUE ))
+
+##assigning moderate acute malnourished (MAM) male 6-59 months old to Male_MAM
+
+Female_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 & age >=6 & age <=59 & oedema == 2 & sex == 2 , na.rm= TRUE ))
+
+##assigning  moderate acute malnourished (MAM) female 6-59 months old to Female_MAM
+
+UM_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 & age >=6 & age <=59 & oedema == 2 & county == "Urban Montserrado" , na.rm= TRUE))
+
+##assigning moderate acute malnourished (MAM) cases children 6-59 months old  in Urban Montserrado to UM_MAM
+
+GB_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 &age >=6 & age <=59 & oedema == 2 & county != "Urban Montserrado" , na.rm= TRUE))
+
+##assigning  moderate acute malnourished (MAM) cases children 6-59 months old other than Urban Montserrado to GB_MAM
+
+MAM <- data.frame(Male_MAM, Female_MAM, UM_MAM, GB_MAM)
```

Comment:
Great that you tried doing this. My main comment here is that what you really want as an output for using `data.frame()` is a table showing the groupings. So, something like:

**sex** | **Urban Montserrado** | **Grand Bassa**
:--- | :--- | :---
Male | &nbsp; | &nbsp;
Female | &nbsp; | &nbsp;

With your code, what this produces is this *"table"*:

```R
  Male_MAM Female_MAM UM_MAM GB_MAM
1       86        149    110    125
```

which shows similar information as the dummy table above but it is not easy for people to understand (i.e., like in a report). Your output is usable (and makes things easier) when performing calculations later on as it combines all the results of the counting you have done but is not as helpful when trying to present results. Try to add more lines of code that will produce a table that looks like the dummy table above in R.

Reply:
Yeah..  Interesting.. will try it.

Reply:
I am reviewing your code response to this now.

I think you were going at a good direction but not entirely what I was meaning.

See what I meant in this discussion - https://github.com/orgs/OxfordIHTM/discussions/8